pulp_ansible/app/urls.py

```diff
@@ -33,6 +33,7 @@
     path(
         "collections/<str:namespace>/<str:name>/versions/<str:version>/",
         GalaxyCollectionVersionDetail.as_view(),
+        name="collection-versions-detail",
```

Comment:
We should probably drop a v2 somewhere in that name. Or we might get into conflicts someday.